webview/src/experiments/util/buildDynamicColumns.tsx

```diff
@@ -58,10 +38,11 @@ const Cell: React.FC<Cell<Experiment, string | number>> = cell => {
   return (
     <Tooltip
       className={tooltipStyles.padded}
-      content={<CellTooltip cell={cell} />}
+      content={String(value)}
```

Comment:
We'll need a component when we display anything more complex, but with this specific implementation for label-less tooltips it's never been less justified to have a separate component.

Reply:
Turns out "anything more complex" also includes adding `stopPropagation` so selection clicks don't register as row clicks.